data.json

```diff
@@ -426,10 +426,10 @@
     "name": "Gatsby",
     "difficulty": "hard",
     "description": "Get a pull request merged into Gatsby and receive a free shirt, socks, or stickers.",
-    "reference": "https://www.gatsbyjs.org/docs/contributor-swag/",
+    "reference": "https://www.gatsbyjs.com/contributing/contributor-swag/",
     "image": "https://i.imgur.com/e7GwaTN.png",
     "dateAdded": "2018-10-24T11:42:09.000Z",
```

Comment:
Please update dateAdded to now please. We might want to rename or add another field for the updated date in the future.